src/plugin.js

```diff
@@ -128,6 +128,12 @@ module.exports = function(app) {
                 minimum: MIN_ZOOM,
                 default: 15,
               },
+              serverType: {
```

Comment:
Is this something that needs to be configured or can it be deduced from the the url?

Reply:
I thought about this but wasn’t sure it was 100% accurate.
What if there are other map types that are locally served or have urls that don’t identify type?

Reply:
Agreed.